bootstrap/controllers/ck8sconfig_controller.go

```diff
@@ -433,18 +459,18 @@ func (r *CK8sConfigReconciler) resolveInPlaceUpgradeRelease(machine *clusterv1.M
 }
 
 // resolveSecretFileContent returns file content fetched from a referenced secret object.
-func (r *CK8sConfigReconciler) resolveSecretFileContent(ctx context.Context, ns string, source bootstrapv1.File) ([]byte, error) {
+func (r *CK8sConfigReconciler) resolveSecretFileContent(ctx context.Context, ns string, source bootstrapv1.FileSource) ([]byte, error) {
```

Comment:
I changed the parameter type from File to FileSource because 1. We did not use any of the fields from File, except ContentFrom. 2. This makes the function reusable for any other types that have a FileSource.